libs/dh/market-participant/actors/data-access-api/src/lib/dh-market-party-b2b-access.store.ts

```diff
@@ -44,7 +44,7 @@ export class DhMarketPartyB2BAccessStore {
     fetchPolicy: 'network-only',
   });
 
-  private credentials: Signal<ActorCredentials | null> = computed(
+  private credentials: Signal<ActorCredentials | null | undefined> = computed(
     () => this.actorCredentialQuery.data()?.actorById.credentials
   );
```

Comment:
question: can this type not be inferred? or do we prefer the explicit typing here? 🙂

Reply:
Have no idea, removed it now :-)